crates/sui-indexer-alt-jsonrpc/Cargo.toml

```diff
@@ -29,6 +29,7 @@ futures.workspace = true
 http.workspace = true
 jsonrpsee = { workspace = true, features = ["macros", "server"] }
 pin-project-lite.workspace = true
+pin-project = "1"
```

Comment:
I thought about adding it to the workspace but figured it's not needed since it's only used by one crate.